subsys/audio_module/audio_module.h

```diff
@@ -308,156 +309,156 @@ int audio_module_open(struct audio_module_parameters *parameters,
  */
 int audio_module_stop(struct audio_module_handle *handle);
 
 /**
  * @brief Send a data buffer to a module, all data is consumed by the module.
  *
- * @param handle[in/out]   The handle for the receiving module instance
- * @param block[in]        Pointer to the audio data block to send to the module
+ * @param handle[in/out]   The handle for the receiving module instance.
+ * @param data[in]         Pointer to the audio data to send to the module.
  * @param response_cb[in]  Pointer to a callback to run when the buffer is
- *                         fully comsumed in a module
+ *                         fully consumed in a module.
  *
- * @return 0 if successful, error otherwise
+ * @return 0 if successful, error otherwise.
  */
-int audio_module_data_tx(struct audio_module_handle *handle, struct audio_data *block,
+int audio_module_data_tx(struct audio_module_handle *handle, struct audio_data *data,
 			 audio_module_response_cb response_cb);
 
 /**
  * @brief Retrieve data from the module.
  *
- * @param handle[in/out]  The handle to the module instance
- * @param block[in]       Pointer to the audio data block from the module
+ * @param handle[in/out]  The handle to the module instance.
+ * @param data[in]        Pointer to the audio data from the module.
  * @param timeout[in]     Non-negative waiting period to wait for operation to complete
  *	                      (in milliseconds). Use K_NO_WAIT to return without waiting,
  *	                      or K_FOREVER to wait as long as necessary.
  *
- * @return 0 if successful, error otherwise
+ * @return 0 if successful, error otherwise.
  */
-int audio_module_data_rx(struct audio_module_handle *handle, struct audio_data *block,
+int audio_module_data_rx(struct audio_module_handle *handle, struct audio_data *data,
 			 k_timeout_t timeout);
 
 /**
- * @brief Send an audio data block to a module and retrieve an audio data block from a module.
+ * @brief Send an audio data to a module and retrieve an audio data from a module.
  *
- * @note The block is processed within the module or sequence of modules. The result is returned
+ * @note The data is processed within the module or sequence of modules. The result is returned
  *       via the module or final module's output FIFO.
  *       All the input data is consumed within the call and thus the input data buffer
- *       maybe released once the function retrurns.
+ *       maybe released once the function returns.
  *
- * @param handle_tx[in]  The handle to the module to send the input block to
- * @param handle_rx[in]  The handle to the module to receive a block from
- * @param block_tx[in]   Pointer to the audio data block to send
- * @param block_rx[out]  Pointer to the audio data block received
+ * @param handle_tx[in]  The handle to the module to send the input data to.
+ * @param handle_rx[in]  The handle to the module to receive data from.
+ * @param data_tx[in]    Pointer to the audio data to send.
```

Comment:
Discuss: Should all the "data" references in this file be renamed to audio_data?